fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/postgres/PostgresResourceNoProcDAO.java

```diff
@@ -385,27 +390,50 @@ public long storeResource(String tablePrefix, List<ExtractedParameterValue> para
             stmt.setLong(1, v_resource_id);
             stmt.setLong(2, v_logical_resource_id);
             stmt.setInt(3, p_version);
-            stmt.setBinaryStream(4, p_payload);
+
+            if (p_payload != null) {
+                stmt.setBinaryStream(4, p_payload);
+            } else {
+                // payload offloaded to another data store
+                stmt.setNull(4, Types.BLOB);
```

Comment:
I think for PostgreSQL the  BYTEA type should be `Types.BINARY`. Not sure if this is handled differently than BLOB.

Reply:
I changed to Types.BINARY to match PostgresResourceDAO which is exercised